docs/source/folder_structure.md

```diff
@@ -1,45 +1,68 @@
-# Folders structure 
-Your project folder containing raw data will be structured in a hierarchical way following BIDS standards:
-```
-project/
-└── subject/
-    └── session/
-        └── datatype/
-            ├── some_raw_data_file
-            ├── some_preprocessed_data_file
-            └── notes
-```
+# Project Folder Structure 
+
+Standardized project folders containing raw data are hierarchically structured according to the 
+[BIDS standard](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html). For example:
 
-Here is an example:
 ```
-my_project/
-└── mouse-01/
-    └── ses-01/
-        └── ephys/
-            ├── my_recording.wav
-            ├── spike_sorted_data.mat
-            └── notes.yaml
+└── project/
+    ├── rawdata/
+    │   └── sub-<label>/
+    │       └── ses-<label>/
+    │           ├── <datatype_1>/
+    │           │   └── raw_data.file
+    │           └── <datatype_2>/
+    │               └── raw_data.file  
+    └── derivatives/
+        └── ...
 ```
-## Naming folders
 
-### Project
-Can have any name, this should be descriptive of the dataset contained in the folder.
+The project folder may have any name descriptive of the project (that does not include spaces). Within the 
+project folder, data is separated into ```rawdata``` and ```derivatives```. ```rawdata``` contains the raw
+data acquired during data collection. ```derivatives``` includes any processed data (e.g. ephys preprocessing, 
+behavioural movement tracking). The subject / session / datatype folder strcture within derivatives should
+match the rawdata folder.
+
+Each subject (e.g. mouse, rat) in the project has a dedicated folder in which all data from 
+experimental sessions are stored.
+In the case of multiple sessions, each session has its own folder where acquired data, belonging to
+a dataype (e.g. ```ephys, behav, imaging, histology``) are stored.
 
-### Subject
-Structure: `sub-<participant label>`
-One folder per subject in this dataset. Labels should be unique for each subject.
-Example: `mouse-01`
+Subject and session folder names cannot contain spaces and must consist of key-value paires separated
+by underscores, e.g. ``sub-001_date-06122022``. 
 
-### Session
-Structure: `ses-<session label>`
+### subject
```

Comment:
The sphinx check action is warning here about going from H1 to H3. I guess it would be nice to ensure we have single level heading increases, e.g. H1 -> H2 -> H3, rather than H1 -> H3.

pinging @niksirbi to see if he has any opinions here.

Reply:
I think it's quite important to have single heading level increases because sphinx uses that information to build the table of contents. I'm happy to see that the sphinx check action is already proving to be useful

Reply:
So, in general for sphinx websites, we should always be very conscious of how we use headings. Each webpage is expected to have one H1 heading. H2 and H3 headings will appear in the TOC, so always keep that in mind

Reply:
thanks I did not know this